docs/docs/02-guide/03-hello-world.md

```diff
@@ -0,0 +1,100 @@
+---
+description: Build your first blockchain and your first Cosmos SDK module.
+title: Hello World
+---
+
+# "Hello world!" Blockchain Tutorial with Ignite CLI
+
+**Introduction**
+
+In this tutorial, you'll build a simple blockchain using Ignite CLI that responds to a custom query with "Hello %s!", where %s is a name provided in the query. 
+This will enhance your understanding of creating custom queries in a Cosmos SDK blockchain.
+
+## Setup and Scaffold
+
+1. **Create a New Blockchain:**
+
+```bash
+ignite scaffold chain hello
+```
+
+2. **Navigate to the Blockchain Directory:**
+
+```bash
+cd hello
+```
+
+## Adding a Custom Query
+
+- **Scaffold the Query:**
+
+```bash
+ignite scaffold query say-hello name --response name
+```
+
+This command generates code for a new query, `say-hello`, which accepts a name, an input and returns it in the response.
+
+- **Understanding the Scaffolded Code:**
+
+	- `proto/hello/hello/query.proto`: Defines the request and response structure.
+	- `x/hello/client/cli/query_say_hello.go`: Contains the CLI commands for the query.
+	- `x/hello/keeper/query_say_hello.go`: Houses the logic for the query response.
+
+
+## Customizing the Query Response
+
+In the Cosmos SDK, queries are requests for information from the blockchain, used to access data like the ledger's current state or transaction details. While the SDK offers several built-in query methods, developers can also craft custom queries for specific data retrieval or complex operations. 
+
+- **Modify `query_say_hello.go`:**
+  
+Update the `SayHello` function in `x/hello/keeper/query_say_hello.go` to return a personalized greeting query.
+
+```go title="x/hello/keeper/query_say_hello.go"
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"hello/x/hello/types"
+	"fmt"
+)
```

Comment:
Optionally it might be a good idea to have some degree of import grouping in code snippets. It adds some semantic value.

For example using three groups, Go standard library imports first, then third party imports and finally app imports:

```suggestion
import (
	"context"
	"fmt"
	
	sdk "github.com/cosmos/cosmos-sdk/types"
	"google.golang.org/grpc/codes"
	"google.golang.org/grpc/status"
	
	"hello/x/hello/types"
)
```

Reply:
That looks good

Reply:
I like it a lot actually. So clean. I've also applied this to the Blog tutorial: https://github.com/ignite/cli/pull/3763

I don't think the Ignite scaffold does this though automatically?

Reply:
> I don't think the Ignite scaffold does this though automatically?

It doesn't but maybe is something we could consider when we migrate to a new template engine

cc @julienrbrt